web-services/query/src/test/java/datawave/webservice/query/logic/TestLegacyBaseQueryLogicTransformer.java

```diff
@@ -1,25 +1,24 @@
 package datawave.webservice.query.logic;
 
-import static org.easymock.EasyMock.expect;
-import static org.easymock.EasyMock.isA;
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.assertNotNull;
-
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-
 import datawave.marking.MarkingFunctions;
 import datawave.webservice.query.cache.ResultsPage;
 import datawave.webservice.query.result.event.EventBase;
 import datawave.webservice.result.BaseQueryResponse;
-
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.powermock.api.easymock.PowerMock;
 import org.powermock.api.easymock.annotation.Mock;
 import org.powermock.modules.junit4.PowerMockRunner;
 
+import java.util.Arrays;
```

Comment:
No changes required in this class

Reply:
Please see above about imports/order

Reply:
My point on this one is that the ONLY changes you have to this class are import reordering.  Changes to this class are completely irrelevant to the pull request and can be reverted.